<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,10,0,0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>17</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Таблиці</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Album">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Albums</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Album_Genre">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Album_Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Alternative_Version">
          <property name="text">
           <string>Alternative_Version</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Artist">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Artist</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Artist_Genre">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Artist_Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Composition">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Composition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Genre">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Genre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Instrumental_Composition">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Instrumental_Composition</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Song">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Song</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="ss">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_add">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QPushButton" name="pb_edit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_delete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="dataTable"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Calculate listen time per song</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="explicit_songs">
          <property name="text">
           <string>Show all explicit Songs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="feat_songs">
          <property name="text">
           <string>Show all songs with feats</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="song_count_per_genre">
          <property name="text">
           <string>Calculate number of song per genre</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
